Gem/Settings/ViewModels/ChainSettingsViewModel.swift

```diff
@@ -46,12 +64,79 @@ class ChainSettingsViewModel: ObservableObject {
 // MARK: - Business logic
 
 extension ChainSettingsViewModel {
-    func getNodes() throws -> [ChainNode] {
-        try nodeService.nodes(for: chain)
+    func fetchNodes() throws {
+        nodes = try nodeService.nodes(for: chain)
+    }
+
+    func fetchNodesMetrics() async {
```

Comment:
```suggestion
    func fetchNodesStatus() async {
```

This is a status of the nodes, metrics have a different context